judge/bridge/judge_handler.py

```diff
@@ -235,6 +242,8 @@ def submit(self, id, problem, language, source):
                 'in-contest': data.contest_no,
                 'attempt-no': data.attempt_no,
                 'user': data.user_id,
+                'file_only': data.file_only,
+                'file_size_limit': data.file_size_limit,
```

Comment:
Please change this to:

```py
                'file-only': data.file_only,
                'file-size-limit': data.file_size_limit,
```

to match the other lines. `-` is automatically converted to `_` at judge.